src/main/resources/io/jenkins/plugins/designlibrary/Dialogs/index.jelly

```diff
@@ -0,0 +1,97 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout" xmlns:s="/lib/samples" xmlns:f="/lib/form">
+  <s:sample>
+    <p class="jdl-leading-paragraph">${%description}</p>
+    <template id="demo-template" data-title="${%Modal}">
+      <div>
+        <h2 class="jenkins-dialog__subtitle">${%Behaviour}</h2>
+        Click on the "X" in the top right or outside to close me.
```

Comment:
can you extract most of the text in this file to properties files please for localisation, see e.g. https://github.com/jenkinsci/design-library-plugin/blob/master/src/main/resources/io/jenkins/plugins/designlibrary/AppBar/index.jelly